api/controllers/MerchStoreController.ts

```diff
@@ -235,4 +238,17 @@ export class MerchStoreController {
       editOrderPickupEventRequest.pickupEvent);
     return { error: null, pickupEvent: pickupEvent.getPublicOrderPickupEvent() };
   }
+
+  @Get('/store/cart')
+  async getCart(@Body() getCartRequest:GetCartRequest, @AuthenticatedUser() user: UserModel): Promise<GetCartResponse> {
```

Comment:
Since this route is a `GET` request, I don't believe we're able to send a body along with the request. Can we either change it so that the UUIDs can be passed as a query parameter, or switch it to a `POST` request? Thanks!

Reply:
Don't think the request really works as a POST request, I think query parameters are likely more accurate - can make the change now